CHANGES.md

```diff
@@ -3,6 +3,19 @@
 Go to https://github.com/metomi/rose/milestones?state=closed
 for a full listing of issues for each release.
 
+<!-- The topmost release date is automatically updated by GitHub Actions. When
+creating a new release entry be sure to copy & paste the span tag with the
+`actions:bind` attribute, which is used by a regex to find the text to be
+updated. Only the first match gets replaced, so it's fine to leave the old
+ones in. -->
+-------------------------------------------------------------------------------
+
+## 2.0b2 (<span actions:bind='release-date'>2021-??-??</span>)
```

Comment:
```suggestion
## 2.0b2 (<span actions:bind='release-date'>YYYY-MM-DD</span>)
```
Would leave you open to being more specific about regex in cylc/release-actions in future.

Reply:
I don't think the regex should be more specific, we can put whatever we want in the span, e.g. "Upcoming" might be more clear